Sources/Sentry/SentryNSDataSwizzling.m

```diff
@@ -0,0 +1,104 @@
+#import "SentryNSDataSwizzling.h"
+#import "SentryNSDataTracker.h"
+#import "SentrySwizzle.h"
+#import <SentryLog.h>
+#import <objc/runtime.h>
+
+@implementation SentryNSDataSwizzling
```

Comment:
Does this also work for `write(to:options:)` of [Data](https://developer.apple.com/documentation/foundation/data/1779858-write)? Do we need to swizzle this as well?

Reply:
Yes it does.